backend/clickhouse/logs.go

```diff
@@ -148,6 +147,7 @@ func (client *Client) ReadLogs(ctx context.Context, projectID int, params modelI
 	}
 	rows.Close()
 
+	span.Finish(tracer.WithError(rows.Err()))
```

Comment:
should we finish after the `getLogsConnection` call? you can change this to
```suggestion
	defer span.Finish(tracer.WithError(rows.Err()))
```